resources/language/resource.language.de_de/strings.po

```diff
@@ -1697,12 +1696,12 @@ msgid "CEC Standby"
 msgstr "CEC Standby"
 
 msgctxt "#32800"
-msgid "Überspringe Intro"
-msgstr ""
+msgid "Skipping intro"
+msgstr "Intro überspringen"
```

Comment:
Hmm, this is wrong. This is the button saying "I'm going to skip the intro", hence "Skipping intro". "Überspringe Intro" is correct in this case. Same with "Überspringe Abspann".

Reply:
Resolved!